tests/test_github.py

```diff
@@ -100,6 +100,18 @@ def test_comment_unmatched_event(self):
         self.waitUntilSettled()
         self.assertEqual(0, len(self.history))
 
+    def test_review_event(self):
+        A = self.fake_github.openFakePullRequest('org/project', 'master', 'A')
+        self.fake_github.emitEvent(A.getReviewAddedEvent('approved'))
+        self.waitUntilSettled()
+        self.assertEqual(3, len(self.history))
```

Comment:
that's a good question. Basically I copied it from the other tests. I think the history is the number of things in the pipeline perhaps?